azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/fragment/setup/components/SetupControlBarViewModel.kt

```diff
@@ -19,50 +21,68 @@ import kotlinx.coroutines.flow.StateFlow
 internal class SetupControlBarViewModel(
     private val dispatch: (Action) -> Unit,
 ) {
-    private lateinit var cameraPermissionStateFlow: MutableStateFlow<PermissionStatus>
-    private lateinit var micPermissionStateFlow: MutableStateFlow<PermissionStatus>
-    private lateinit var cameraStateFlow: MutableStateFlow<CameraState>
+    private lateinit var cameraIsEnabledStateFlow: MutableStateFlow<Boolean>
+    private lateinit var micIsEnabledStateFlow: MutableStateFlow<Boolean>
+    private lateinit var deviceIsEnabledStateFlow: MutableStateFlow<Boolean>
+
+    private lateinit var visibleStateFlow: MutableStateFlow<Boolean>
```

Comment:
I think we can indicate about, what is visible?

Reply:
The whole control. This is a visible property Control bar view itself, to a particular internal button